.ci/setup_env_ubuntu.sh

```diff
@@ -45,7 +45,7 @@ chronic sudo -E apt install -y build-essential python pkg-config zlib1g-dev
 echo -e "Install CRI-O dependencies available for Ubuntu $VERSION_ID"
 chronic sudo -E apt install -y libdevmapper-dev btrfs-tools util-linux
 
-if [ "$VERSION_ID" == "16.04" ]; then
+if [ "$VERSION_ID" == "16.04" ] && [ "$(arch)" != "ppc64le" ]; then
 	echo "Install os-tree"
```

Comment:
not your doing I think, but could you lift the echo outside the `if`, as I think `os-tree` will be installed by the next step, no matter what... thx!

Reply:
Done.